openlibrary/plugins/upstream/models.py

```diff
@@ -822,17 +822,6 @@ class MergeAuthors(Changeset):
     def can_undo(self):
         return self.get_undo_changeset() is None
 
-    def process_docs_before_undo(self, docs):
```

Comment:
Is this called anywhere? Do we need to remove calls to it?

Reply:
@mek It is a hook called by the parent class, but I don't think it does anything useful, in fact following author redirects here is the _worst_ thing it could do. The operation is to revert an author merge, and this looks at the undo changeset, and replaces the original author (the one we are trying to revert back to) with the current target of the redirect, which is the incorrect author we are trying to revert away from. This is why author merges never seemed to work, this code ensured that the incorrect author stayed attached to all the works.

I think it has been expressed by other too (@cdrini and @tfmorris ?) that the revert of a changeset should be a simple undo, with no extra unexpected logic.